bevy_ios_safearea/src/plugin.rs

```diff
@@ -54,12 +64,13 @@ fn init(
 
             let top = unsafe { crate::native::swift_safearea_top(ui_view) };
             let bottom = unsafe { crate::native::swift_safearea_bottom(ui_view) };
+            let left = unsafe { crate::native::swift_safearea_left(ui_view) };
+            let right = unsafe { crate::native::swift_safearea_right(ui_view) };
 
-            let safe_area = IosSafeArea { top, bottom };
+            let safe_area = IosSafeArea { top, bottom, left, right, };
 
+            commands.insert_resource(safe_area);
```

Comment:
how does insert-overwrite behave with change-detection?

Reply:
I think it is treated as `added` and `changed` but I don't remember now.